docs/guides/1.23.0-upgrade-guide.md

```diff
@@ -0,0 +1,25 @@
+---
+page_title: "Upgrade Guide 1.23.0"
+---
+
+# MongoDB Atlas Provider 1.23.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.23.0 has a number of new and exciting features.
+
+## New Resources, Data Sources, and Features
+
+- You can now leverage auto-scaling to scale the instance size of each individual shard independently for sharded and geo-sharded clusters defined with `mongodbatlas_advanced_cluster`. To learn more, see the [Advanced Cluster New Sharding Configurations - Use Auto-Scaling Per Shard](advanced-cluster-new-sharding-schema#use-auto-scaling-per-shard).
+
+- You can now pin a cluster's FCV before upgrading in Atlas, which enables you to revert MongoDB to the previous version after upgrading without experiencing version compatibility issues. This is supported in `mongodbatlas_advanced_cluster` and `mongodbatlas_cluster` using `pinned_fcv`. To learn more, please review either [mongodbatlas_advanced_cluster](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_cluster) or [mongodbatlas_cluster](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cluster) resource documentation.
+
+- You can now configure the default time limit in milliseconds for individual read operations to complete in `mongodbatlas_advanced_cluster` using `advanced_configuration.0.default_max_time_ms`. To learn more, please review [mongodbatlas_advanced_cluster](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_cluster) resource documentation.
+
+
```

Comment:
I would keep the "Terraform MongoDB Atlas modules" section as in previous guides

Reply:
it was commented in the previous guides PR. maybe yo can confirm with Zuhair